.github/workflows/update_aas_extension.yml

```diff
@@ -104,6 +104,7 @@ jobs:
             aasStatsName="dd-dotnet-latest-build-stats"
             aasProfilerOnlyName="dd-dotnet-latest-build-profiler-only"
             aasProfilerBackendName="dd-dotnet-profiler-backend-test-latest-build"
+            aasSecuritySample="security-samples-aspnetcore"
```

Comment:
🙏 I feel like we should have dotnet somewhere in the app name so we can find it more easily... how was is it to change? 😬

Reply:
changed to `dd-dotnet-security-aspnetcore` ?